src/settings/FundTypeSettings/FundTypeSettings.js

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  FormattedMessage,
+  injectIntl,
+  intlShape,
+} from 'react-intl';
+import { Field } from 'redux-form';
+
+import { ControlledVocab } from '@folio/stripes/smart-components';
+import {
+  TextArea,
+  TextField,
+} from '@folio/stripes/components';
+
+import { FUND_TYPES_API } from '../../common/const';
+import { fundTypesResource } from '../../common/resources';
+
+const fieldComponents = {
+  // eslint-disable-next-line react/prop-types
+  name: ({ fieldProps }) => {
+    return (
+      <Field
+        {...fieldProps}
+        component={TextField}
+        fullWidth
+      />
+    );
+  },
+  // eslint-disable-next-line react/prop-types
+  description: ({ fieldProps }) => {
+    return (
+      <Field
+        {...fieldProps}
+        component={TextArea}
+        fullWidth
+      />
+    );
+  },
+};
+
+const hiddenFields = ['numberOfObjects', 'lastUpdated'];
+const visibleFields = ['name'];
+const columnMapping = {
+  name: <FormattedMessage id="ui-finance.fundType.name" />,
+};
+
+const fundTypeLabel = <FormattedMessage id="ui-finance.fundType.label" />;
+
+const FundTypeSettings = ({
+  intl,
+  stripes,
+  resources,
+  mutator,
+}) => {
+  return (
+    <ControlledVocab
+      stripes={stripes}
+      resources={resources}
+      mutator={mutator}
+      dataKey={undefined}
```

Comment:
shiiiiiiiit... 
![image](https://user-images.githubusercontent.com/43577432/69138834-745dc880-0ad0-11ea-90a6-7be6c5cb1431.png)

Reply:
hm, don't see any issues without dataKey now, mb it was fixed